### DISCUSSION TITLE
Percent

### DISCUSSION THREAD

**Discussion by SuperStark** [microsoft/winget-cli#3488] [Q&A] [answered]
Hi,

Probably a quick one, can we show percentage during download instead of the progress bar?
eg: 30% instead of █████████████ 3.8mb 6mb

**Comment by denelon** [comment, marked as answer]
The progress bar renders the download visually as a percentage.

You could create a new issue with a feature to make this a setting or an argument to render a percentage rather than the size.

**Reply by SuperStark** [reply]
Makes sense, I've coded in a check for how many bars there are to determine the percentage currently, I didn't use the MB (size) part as this has no consistency, sometimes it only shows the progress bar which seems like bug (or it doesn't know the size?)

Either way, I'll raise a request for a percent switch/argument
